Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -132,16 +170,55 @@ namespace MultiplayerSample
         GetEntity()->GetTransform()->SetLocalRotationQuaternion(newOrientation);
 
         // Update velocity
-        UpdateVelocity(*playerInput);
+        UpdateVelocity(*playerInput, deltaTime);
 
-        GetNetworkCharacterComponentController()->TryMoveWithVelocity(GetVelocity(), deltaTime);
+        // absolute velocity is based on velocity generated by the player and other sources
+        const AZ::Vector3 absoluteVelocity = GetBaseVelocity() + GetLocalVelocity();
+
+        // if we're not moving down on a platform and have a negative velocity we're falling
+        GetNetworkAnimationComponentController()->ModifyActiveAnimStates().SetBit(
+            aznumeric_cast<uint32_t>(CharacterAnimState::Falling), !onGround && absoluteVelocity.GetZ() < 0.f);
+
+        GetNetworkCharacterComponentController()->TryMoveWithVelocity(absoluteVelocity, deltaTime);
+
+        m_wasOnGround = onGround;
     }
 
-    void NetworkPlayerMovementComponentController::UpdateVelocity(const NetworkPlayerMovementComponentNetworkInput& playerInput)
+    void NetworkPlayerMovementComponentController::UpdateVelocity(const NetworkPlayerMovementComponentNetworkInput& playerInput, float deltaTime)
```

Comment:
Should be renamed to UpdateSelfGeneratedVelocity?

Reply:
This updates both velocities currently